src/apps/viewer/index.ts

```diff
@@ -117,6 +118,7 @@ export class Viewer {
             config: [
                 [PluginConfig.General.DisableAntialiasing, o.disableAntialiasing],
                 [PluginConfig.General.PixelScale, o.pixelScale],
+                [PluginConfig.General.EnableWboit, o.enableWboit],
```

Comment:
Could the feature be enabled/disabled during runtime? And if so, do you think it would be better to have this is a canvas/renderer prop?